kiwi/boot/functions.sh

```diff
@@ -5517,6 +5517,21 @@ function killShell {
     fi
 }
 #======================================
+# getDeviceTransportType
+#--------------------------------------
+function getDeviceTransportType {
+    # /.../
+    # function to get the transport type of the
+    # given device. It uses lsblk command.
+    # ----
+    local device=$1
+    if [ -z "$device" ] || [ ! -e "$device" ];then
+        echo 1 ; return 1;
+    fi
+    local disk=${device%%[0-9]*}
+    echo "`lsblk -dno tran $disk`"
```

Comment:
could you use the $() syntax

```bash
echo $(lsblk -dno tran $disk)
```

I switched to this invocation type at some point in the past, there are still some backticks call left but I'd like to get rid of them. Thanks